axis_utils/axis_utils2.F90

```diff
@@ -55,11 +56,44 @@ module axis_utils2_mod
   !! @param [inout] data2 Interpolated data
   !! @param method Either "linear" or "cubic_spline" interpolation method, default="linear"
   !! @ingroup axis_utils2_mod
```

Comment:
This comment block should be above the interface it was documenting (interp_1d i think)

Reply:
This seems like it could be misplaced, but I just suspected it was further explanation of the module

Reply:
Doxygen looks where it is in the code to figure out where the comments end up so it'll need to be above what it's documenting to be included correctly. If its not above/next to something or in a routine it'll usually get skipped or added somewhere random.

We'll probably need to add more doxygen changes eventually for mixed precision in general, so I just don't want to lose any of the old comments or get things mixed up down the line.